docs/sources/v0.47.x/testing-guides/injecting-faults-with-xk6-disruptor/first-steps.md

```diff
@@ -2,30 +2,212 @@
 title: 'xk6-disruptor first steps'
 excerpt: 'xk6-disruptor is a k6 extension providing fault injection capabilities to k6.'
 weight: 01
+aliases:
+  - /docs/k6/latest/javascript-api/xk6-disruptor/get-started/ 
 ---
 
-# xk6-disruptor first steps
+# First steps
 
-[xk6-disruptor](https://github.com/grafana/xk6-disruptor) is an extension that adds fault injection capabilities to k6.
+This document will guide you through running your first fault injection test using xk6-disruptor, while providing pointers to the more detailed documentation sections along the way. We will cover the following sections:
 
-It provides a Javascript [API](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/xk6-disruptor/) to inject [faults](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/xk6-disruptor/faults) such as errors and delays into HTTP and gRPC requests served by selected Kubernetes [Pods](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/xk6-disruptor/poddisruptor) or [Services](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/xk6-disruptor/servicedisruptor).
+1. [Checking requirements](#checking-requirements)
+1. [Installing xk6-disruptor](#installing-xk6-disruptor)
+1. [Creating a simple k6 test](#creating-a-simple-k6-test)
+1. [Add fault injeciton capabilities](#add-fault-injection-capabilities)
 
+# Checking requirements
+
+xk6-disruptor focuses on injecting faults on applications running in Kubernetes. If you already have your application deployed to a development Kubernetes cluster, you are all set! If you don't, we will guide you through deploying a demo application on a local `minikube` cluster.
+
+In order to follow this guide, you will need
+
+- A microservice-based application that communicates over HTTP or GRPC.
+- The ability to reach at least one service exposed by the application from the machine where you will run the tests.
+- Privileged access to the Kubernetes cluster where your application is running from the machine where you will run the tests.
+
+## Use our demo application
```

Comment:
I think this section should go in its own high-level section, before the "creating a simple test" section.